app/controllers/facility_order_details_controller.rb

```diff
@@ -14,9 +14,20 @@ def initialize
   end
 
   def show
-    # This is deprecated in favor of OrderManagement::OrderDetailsController, but
-    # we want to avoid 404s
-    redirect_to facility_order_path(current_facility, @order)
+    respond_to do |format|
+      format.html do
+        # This is deprecated in favor of OrderManagement::OrderDetailsController, but
+        # we want to avoid 404s
+        redirect_to facility_order_path(current_facility, @order)
+      end
+
+      format.ics do
+        calendar = ReservationCalendar.new(@order_detail.reservation)
```

Comment:
Right now, it'd error if the reservation is `nil`, I suspect that's okay at the moment